### DISCUSSION TITLE
Does the GPU or number of CPU threads matter?

### DISCUSSION THREAD

**Discussion by kd4e** [blakeblackshear/frigate#9845] [Ask A Question] [answered]
I'm looking at a dedicated computer for Frigate (plus openHAB). I'm comparing a Dell Optiplex 7050 with an Intel i5-7500 (4 cores, 4 threads, HD Graphics 630) **to** a 9020 with an Intel i7-4770 (4 cores, 8 threads, GPU HD Graphics 4600). Either one would have at least 24GB ram, perhaps 32GB. (I'll be hosting 10-12 cameras.) Will I notice any difference? Should I be looking at a newer CPU or a different (perhaps add-on) GPU? (I'm hoping to use Deepstack or Tensorflow rather than Coral. Please also include consideration of potential added processing demands in the new Frigate version due this summer.) Thanks.

**Comment by NickM-27** [comment]
If you used the 7500 then you could use openvino instead of a coral

**Reply by kd4e** [reply]
At first I thought that was going to be a joke about this all causing me to take up drinking wine!  lol
A Search led me here: [https://viso.ai/computer-vision/intel-openvino-toolkit-overview/]
I'd never heard of it.
It's unclear to me how it might integrate with Frigate, though I see that it plays nicely with TensorFlow.
It's also unclear if it works - without requiring access to the Internet.
Have you tried it, yet?
Also, why would only the i5-7500 work and not the i7-4770? (I just read this "6th to 12th generation Intel® Core™ processors and Intel® Xeon® processors")
Thanks!

**Reply by NickM-27** [reply, marked as answer]
OpenVino is covered in the frigate docs https://docs.frigate.video/configuration/object_detectors#openvino-detector

**Reply by kd4e** [reply]
Way cool on all! (I had to return the just delivered Dell 9020 w/i7-4770, due to a bad motherboard, but already had two 7050's.)
I'll load Frigate and begin the hands-on learning process.
Thanks!
